js/formats/woz.ts

```diff
@@ -22,7 +22,7 @@ function stringFromBytes(data: DataView, start: number, end: number): string {
     return String.fromCharCode.apply(
         null,
         new Uint8Array(data.buffer.slice(data.byteOffset + start, data.byteOffset + end))
-    );
+    ) as string;
```

Comment:
Weird that the type of `apply()` can't be inferred. Maybe they'll get to that in a later release.